CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+We follow [Semantic Versioning](http://semver.org/) as a way of measuring stability of an update. This
+means we will never make a backwards-incompatible change within a major version of the project.
+
+## v3.0.0
```

Comment:
Ah I see. The first thing to note is that this is a totally separate (albeit very related) project to `tor`. We can track versions independently here and rely on pip to resolve dependencies between versions.

While I agree it's a good argument for updating `tor`, this is a separate project and need not follow the same rule of backwards-compatibility. I could see it two ways:

1. Consider `tor` and associated libraries as one "package" release, tracking to the same major version for all libraries. If `tor` increments to 4.0 later, this library would too. **Example:** [Rails](https://github.com/rails/rails/blob/master/rails.gemspec#L21-L29)
2. Consider the libraries a separate project from `tor`, tracking with their own pre-1.0 version until the library API stabilizes. **Example:** [praw](https://github.com/praw-dev/praw/blob/master/setup.py#L40-L41)